tests/testthat/curly-curly/mixed-out.R

```diff
@@ -59,12 +59,34 @@ call({{ x }})
 
 ##  ............................................................................
 ##  multiple                                                                ####
-call({
+call("test", {
   1
-}, a + b, {
-  33 / f(c)
 })
 
+call(
+  "test", {
```

Comment:
So the newline isn't added here because it's the last argument? Can we easily check if the brace is on the same line as the opening parenthesis of the function call?

Reply:
Not easily but we can in `eaac448d22ec000d99030a367852c5bae9b6ceb5` (pushed to master). :-)